app/services/irs_attempts_api/redis_client.rb

```diff
@@ -30,7 +30,7 @@ def read_events(count = 1000)
     end
 
     def self.clear_attempts!
-      if !Rails.env.test?
+      unless ['test', 'development'].include?(Rails.env)
```

Comment:
```suggestion
      unless %w(test development).include?(Rails.env)
```

I got sick of not being able to do this in dev.

Rubocop got sick of me and wants it to be `%w()`